CSS/Layout/layout.css

```diff
@@ -0,0 +1,45 @@
+.head {
+  height: 4rem;
+  padding: var(--sp-md);
+  box-shadow: 2px 0 2px var(--gray-600);
+  /* background-color: var(--gray-200); */
+}
+.logo {
+  gap: var(--sp-xsm);
+  font-size: var(--fs-xlg);
+  font-weight: bold;
+  color: var(--primary);
+}
+.nav {
+  font-size: large;
+  flex-grow: 0.1;
```

Comment:
Is it a typo?? I don't think `flex-grow:0.1;` is significant. If you haven't given the `flex-grow` property to any of its siblings, the `flex-grow:0.1 is same as flex-grow:1 or any other value`

Reply:
Have given the flex-grow prop to that element so that it can grow over left over space and it's child elements are aligned freely i.e not stacked.

before:
![image](https://user-images.githubusercontent.com/91076362/153124737-5d8c285c-e868-48f6-a1a9-fafa855446ce.png)

After:
![image](https://user-images.githubusercontent.com/91076362/153124654-a5d71d8c-e8bc-4ecd-80bf-4fc758773a7c.png)

Reply:
Ya, I know the flex grow property is needed here, but I'm just saying that `flex-grow:0.1` and `flex-grow:1` are the same in this case, so its better to write `flex-grow:1` to avoid any confusion

Reply:
Anyways, its not wrong but I just suggested the change and its not compulsory, so I'm approving it👍